src/app/Library/CrudPanel/Traits/Update.php

```diff
@@ -26,6 +26,8 @@ public function update($id, $data)
         $data = $this->compactFakeFields($data);
         $item = $this->model->findOrFail($id);
 
+        $data = $this->changeBelongsToNamesFromRelationshipToForeignKey($data, $this->getFields());
```

Comment:
Yeah so I've tested it... it works... but I don't understand _why_ it works, since the method itself only has one argument 😅 Eliminated that argument here and in Create... still works.

What do you think @pxpm ?

Reply:
yes it works, sorry, I started by using the second parameter, refactored the function and didn't removed the extra parameter.

Indeed it works even with second parameter. It would only fail if the parameter was required and not passed, otherwise is just ignored.

I've just pushed a fix for both concerns you raised.